administration/src/components/generation/AddEakForm.css

```diff
@@ -0,0 +1,18 @@
+.add-eak-form-label {
```

Comment:
Should we maybe use styled-components? What do you think @Schedulaar ?

Reply:
Yeah, would be nice, I think